forge-gui/src/main/java/forge/player/HumanPlaySpellAbility.java

```diff
@@ -264,7 +264,10 @@ private boolean announceValuesLikeX() {
         if (needX) {
             if (cost.hasXInAnyCostPart()) {
                 final String sVar = ability.getSVar("X"); //only prompt for new X value if card doesn't determine it another way
-                if ("Count$xPaid".equals(sVar) || sVar.isEmpty()) {
+                // check if X != 0 is even allowed or the X shard got removed
+                boolean replacedXshard = ability.isSpell() && ability.getHostCard().getManaCost().countX() > 0 &&
```

Comment:
@Hanmac 
this fixed casting _Aether Tide_ via _Jodah, Archmage Eternal_ and still choosing X
that shouldn't be allowed?

Reply:
ohm _Aether Tide_ still has X in the Additional Cost ...

but i will need to ask Judges about this

Reply:
Rules currently say that X must be 0 for _Aether Tide_

> 107.3b If a player is casting a spell that has an {X} in its mana cost, the value of X isn’t defined by the text of that spell, and an effect lets that player cast that spell while paying neither its mana cost nor an alternative cost that includes X, then the only legal choice for X is 0. This doesn’t apply to effects that only reduce a cost, even if they reduce it to zero. See rule 601, “Casting Spells.”

so while X is in the Additional Cost, it still fails because the ManaCost X says it's Zero